tests/dp_engine_test.py

```diff
@@ -363,7 +362,7 @@ def test_aggregate_report(self, mock_create_accumulator_factories_function):
             "Differentially private: Computing <Metrics: ['privacy_id_count', 'count', 'mean']>"
             "\n1. Per-partition contribution bounding: randomly selected not more than 2 contributions"
             "\n2. Cross-partition contribution bounding: randomly selected not more than 3 partitions per user"
-            "\n3. Private Partition selection: using Truncated Geometric method with (eps= 0.5, delta = 5e-11)"
+            "\n3. Private Partition selection: using Truncated Geometric method with (eps= 0.1111111111111111, delta = 1.1111111111111111e-11)"
```

Comment:
Do you know why this has changed?

Reply:
Yes, previously there was no aggregation, because it was mocked. Now we don't need to aggregate, that's why real combiner was created and it requests for budget, that's why less budget on partition selection